tdrs-backend/docs/api/authentication.md

```diff
@@ -4,3 +4,14 @@ For clients to authenticate, they have to authenticate with Login.gov via the ba
  This will allow the backend to identify the browser which requested access and authorize them based on the cookie they provide in their API calls. 
 
  The secured portion of this authorization is due to the httpOnly cookie being inaccessible to the client's local browser. 
+
+# Generating API token
+
+In order to use APIs, the `OFA Admin` user has to generate a new token and use it in the API request following these steps:
```

Comment:
@raftmsohani ofa admin or ofa system admin? both?

Reply:
It is only for OFA SYS ADMIN. Do you want ofa admin to have access too?

Reply:
ofa system admin is fine with me. just need to update this markdown with the relevant permission group.

Reply:
Done!